### ISSUE TITLE
[BUG] Ares causes Baritone to not save settings

### ISSUE THREAD

**Issue by SolsticeSpectrum** [AresClient/ares#35]
When you remove Fabritone fron fabric.mod.json and then from the jars folder and replace it in mods folder with baritone 1.6.3 which has official Fabric support, it causes issues when trying to change settings like freeLook, renderGoal and others. Also I would probably switch to the official one since Fabritone is now obsolete. Here is a build.gradle I came up with to integrate the official one.

```gradle
    maven {
        url 'https://jitpack.io'
    }
    maven {
        url 'https://repo.spongepowered.org/repository/maven-public/'
    }

    modImplementation ('com.github.cabaletta:baritone:v1.6.3') {
        exclude(group: "org.lwjgl.lwjgl", module: "lwjgl-platform")
        exclude(group: "org.lwjgl.lwjgl", module: "lwjgl")
    }

    include 'com.github.cabaletta:baritone:v1.6.3'
```

**Comment by ghost** [comment]
Were you using the baritone API package? That is what Ares (and as far as I know, all other clients) use to interface with baritone

**Comment by Tigermouthbear** [comment]
this is fixed in the 1.17 branch, will be merged when 1.17 is done
